assert-camunda-git-emails/action.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Assert Camunda Git Emails
+
+description: Asserts that only Camunda email addresses are used in Git commits.
+
+inputs: {}
+
+runs:
+  using: composite
+  steps:
+  - uses: actions/checkout@v2
+    with:
+      fetch-depth: 0
+      ref: ${{ github.event.pull_request.head.sha }}
+
+  - name: Check for violating email addresses
+    shell: bash
+    env:
+      GIT_RANGE: "origin/${{ github.base_ref }}...${{ github.event.pull_request.head.sha }}"
```

Comment:
would it make sense to do safety check in the action and do nothing if the action is run outside of the PR context?
it case `on: [pull_request]` is omitted.